buildscripts/pages/gallery.html

```diff
@@ -84,10 +84,16 @@ <h1 class="d-none d-lg-block">udStream Showcase</h1>
 				description: "This shows how you can display underground data."
 			},
 			{
-				name: "South-East Queensland",
+				name: "SE Queensland",
```

Comment:
`SE` means nothing to anyone, you're better off removing it entirely and just having "Queensland"? "SEQ" means something to Queenslanders but not the rest of the world.

Reply:
Reverted back to South-East Queensland as above updates fixed responsiveness issues.